test/OrchardCore.Commerce.Tests.UI/.htmlvalidate.json

```diff
@@ -0,0 +1,18 @@
+﻿{
```

Comment:
Yes, we need it. It's different form the default because it also has `"form-dup-name": "off"`. This is required because this rule it doesn't work with the stock OC checkbox type `BooleanField`s. They have both the normal input and a hidden field with the same name and a default value which violates `form-dup-name` but not in a way that matters.

Since we use the NuGet package, I don't think we can reference that default config via "extends" because you don't know the path, so I just copied the whole thing into this config.